cx_Freeze/macdist.py

```diff
@@ -165,6 +171,7 @@ def initialize_options(self):
         self.iconfile = None
         self.qt_menu_nib = False
         self.bundle_name = self.distribution.get_fullname()
+        self.plist_items = None
```

Comment:
This element is a list, see https://github.com/marcelotduarte/cx_Freeze/blob/master/cx_Freeze/dist.py#L186